mailpoet/tests/acceptance/Segments/ManageSegmentsCest.php

```diff
@@ -425,7 +425,7 @@ public function createSegmentFromTemplate(\AcceptanceTester $i) {
     $i->waitForElement('.mailpoet-templates-card-grid div:first-child');
     $i->click('.mailpoet-templates-card-grid div:first-child');
     $i->waitForText('Recently Subscribed');
-    $i->click('Back');
+    $i->click('#mailpoet-segments-back-button');
```

Comment:
I'd be curious to hear more about why do you prefer to use the text instead of IDs? Is it to make sure that the interface is displaying what we expect it to display? Maybe it is even a QA best practice?

Here, I opted to use the ID instead of the text to avoid a scenario where we change the text, push the commits to GitHub only to realize that we broke an acceptance test. The test would be broken, but the interface would actually be working. At the end of the day, it is a trade-off, and I understand if the recommendation is to use the text instead of the ID.